test/evp_kdf_test.c

```diff
@@ -160,6 +162,180 @@ static int test_kdf_pbkdf2(void)
     return ret;
 }
 
+#ifndef OPENSSL_NO_ARGON2
+static int test_kdf_argon2(void)
+{
+    int ret;
+    EVP_KDF_CTX *kctx;
+    OSSL_PARAM params1[3], params2[5], params3[6], params4[5], *p;
```

Comment:
why use seperate arrays?

Reply:
Not relevant anymore, since the tests were separated into distinct functions.